test/helm/suite_test.go

```diff
@@ -27,13 +26,11 @@ func init() {
 	imageName = flag.String("image-name", "", "Name of the locally built for testing Jenkins Operator Image.")
 }
 
-func TestHelm(t *testing.T) {
-	RegisterFailHandler(Fail)
+// func TestHelm(t *testing.T) {
```

Comment:
we need to fix this test as well, again we can do it in another PR, no need to address it now